sdk/appconfiguration/azure-appconfiguration-provider/samples/refresh_sample.py

```diff
@@ -0,0 +1,65 @@
+# ------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from collections import namedtuple
+from azure.appconfiguration.provider import load
+from azure.appconfiguration import (
+    AzureAppConfigurationClient,
+    ConfigurationSetting,
+)
+import os
+import time
+
+connection_string = os.environ.get("AZURE_APPCONFIG_CONNECTION_STRING")
+print(connection_string)
+
+# Setting up a configuration setting with a known value
+client = AzureAppConfigurationClient.from_connection_string(connection_string)
+
+configuration_setting = ConfigurationSetting(key="message", value="Hello World!")
+
+client.set_configuration_setting(configuration_setting=configuration_setting)
+
+
+def my_callback():
+    print("Refreshed!")
+
+
+def my_callback_on_fail():
+    print("Refresh failed!")
+
+
+# Connecting to Azure App Configuration using connection string
+RefreshItem = namedtuple("RefreshItem", ["key", "label"], defaults=[None, "\0"])
```

Comment:
If we want to add this into the API - we could give it a more generic name. We probably also only want to have a default for `label`, not `key`:
```python
class SettingKeyLabel(NamedTuple):
    key: str
    label: str = "\0"
```